README.md

```diff
@@ -1,4 +1,4 @@
-![Kuma](https://kuma.io/images/brand/kuma-logo-new.svg)
+![Kuma](https://kuma-public-assets.s3.amazonaws.com/kuma-logo-v2.png)
```

Comment:
nitpick: you can run optipng to optimize this png:

```
$ optipng kuma-logo-v2.png                                                                                                                                                                                                                                       -- INSERT --
** Processing: kuma-logo-v2.png
1765x637 pixels, 4x8 bits/pixel, RGB+alpha
Reducing image to 8 bits/pixel, 17 colors (16 transparent) in palette
Input IDAT size = 18684 bytes
Input file size = 18786 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0		IDAT size = 9256

Selecting parameters:
  zc = 9  zm = 8  zs = 0  f = 0		IDAT size = 9256

Output IDAT size = 9256 bytes (9428 bytes decrease)
Output file size = 9425 bytes (9361 bytes = 49.83% decrease)
```

Reply:
<img width="883" alt="kuma-logo-v2" src="https://user-images.githubusercontent.com/2753650/212854298-143915aa-e056-4a79-82f1-b539db60fcee.png">